checker-util/src/main/java/org/checkerframework/checker/regex/util/RegexUtil.java

```diff
@@ -195,7 +195,9 @@ public static boolean isRegex(String s, int groups, int... nonNullGroups) {
         List<Integer> computedNonNullGroups = getNonNullGroups(p.pattern(), getGroupCount(p));
         if (groups <= getGroupCount(p)) {
             ArrayList<Integer> paramNonNullGroups = new ArrayList<>(nonNullGroups.length);
```

Comment:
I think it is inefficient to create a list for no purpose other than to call `containsAll`.  It's more efficient, and the code is no longer, to iterate over the array checking each element.